pypasta/Repository/Mbox.py

```diff
@@ -488,6 +488,9 @@ def __getitem__(self, message_id):
 
         return ret
 
+    def get_patchwork_id(self, message_id):
```

Comment:
must be get_patchwork_ids

Reply:
For a particular Patchwork project, a message id will always map to a single patchwork id

Reply:
What if two different messages with the same message id will be sent to a list? However, there is a chance that the case could happen, and we wouldn't treat it as we should.

Reply:
Afaik (project_id, message_id) is guaranteed to be unique in patchwork. They ignore subsequent messages to the same project with the same message_id iirc